pkgs/development/ocaml-modules/ocaml-syntax-shims/default.nix

```diff
@@ -11,6 +9,8 @@ buildDunePackage rec {
     sha256 = "1j7848khli4p7j8i2kmnvhdnhcwhy3zgdpf5ds5ic30ax69y3cl9";
   };
 
+  minimalOCamlVersion = "4.02.3";
+
   useDune2 = true;
```

Comment:
```suggestion
```

Since you are causing a large rebuild, you should clean a bit more (remove this noisy `useDune2`). Also, I’m not sure it is interesting to set `minimaOCamlVersion` to a value that is already the minimal version supported by `buildDunePackage`.

Reply:
To be clear, you want me to also remove `useDune2` references in this PR?

Reply:
No, just this one. The other files can be fixed at very little cost.